### DISCUSSION TITLE
请问怎么依赖vcpkg中某个库的依赖？

### DISCUSSION THREAD

**Discussion by chen-qingyu** [xmake-io/xmake#4493] [Q&A]
比如我在`xmake.lua`中添加`add_requires("vcpkg::osgearth")`，但是这样报错`error LNK2001: unresolved external symbol xxx`，因为osgearth依赖osg，osg依赖gdal等一大堆库，其中一些库又依赖一大堆库，怎么自动依赖呢？比如只需要`add_requires("vcpkg::osgearth")`然后设置某个选项就行了？

**Comment by waruqi** [comment]
目前只有使用 xmake-repo 官方仓库的包 才支持级联依赖，vcpkg 的，你只能外面自己配
